src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.GenerateAssemblyInfo.targets

```diff
@@ -173,11 +178,18 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <PropertyGroup>
     <!-- Generate attributes in the main assembly if we're targeting a C# project and using the Razor Sdk. -->
-    <CoreCompileDependsOn Condition="'$(ResolvedRazorCompileToolset)'=='RazorSdk' and '$(Language)' == 'C#' and '$(GenerateRazorAssemblyInfo)' == 'true'">
-      $(CoreCompileDependsOn);_GenerateRazorAssemblyInfo
+    <CoreCompileDependsOn Condition="'$(ResolvedRazorCompileToolset)'=='RazorSdk' and '$(Language)' == 'C#'">
+      $(CoreCompileDependsOn);
+      _GenerateRazorAssemblyInfo
     </CoreCompileDependsOn>
 
-    <_GenerateRazorAssemblyInfoDependsOn>PrepareForBuild;_CoreGenerateRazorAssemblyInfo</_GenerateRazorAssemblyInfoDependsOn>
+    <_GenerateRazorAssemblyInfoDependsOn>RazorGetAssemblyAttributes;PrepareForBuild;_CoreGenerateRazorAssemblyInfo</_GenerateRazorAssemblyInfoDependsOn>
+
+    <!-- In 3.0 or later, we need to invoke a target to determine MVC specific configuration. -->
```

Comment:
Will VS and friend IDEs have problems with the new targetified configuration req? I'm pretty positive the answer is no since you're including the target in a dependency chain and it isn't responsible for design time vital information; but should definitely check VS to make sure things work as expected.

Reply:
Not that I know of. All of these control what gets attributes get generated when the app's assembly or views assembly get compiled, which shouldn't happen as part of design time build